Check jobs (Pytest) failing on Python 3.6 and 3.7

A single test is failing on Python 3.6 and 3.7, but succeeding on 3.5, 3.8 and 3.9.

```
    def test_desc_from_uri():
        URI = "x-sonos-http:track%3a3402413.mp3?sid=2&amp;flags=32&amp;sn=1"
>       assert desc_from_uri(URI) == "SA_RINCON2311_12345678"
E       AssertionError: assert 'RINCON_AssociatedZPUDN' == 'SA_RINCON2311_12345678'
E         - SA_RINCON2311_12345678
E         + RINCON_AssociatedZPUDN

tests/test_musicservices.py:323: AssertionError

```

--- a/soco/music_services/music_service.py
+++ b/soco/music_services/music_service.py
@@ -838,7 +838,10 @@ def desc_from_uri(uri):
     # the uri as if it were http
     if ":" in uri:
         _, uri = uri.split(":", 1)
-    query_string = parse_qs(urlparse(uri, "http").query)
+    query_string_initial = parse_qs(urlparse(uri, "http").query)
+    # Strip 'amp;' prefix from keys if required
+    #   See https://github.com/SoCo/SoCo/issues/810
+    query_string = {k.replace("amp;", ""): v for k, v in query_string_initial.items()}
     # Is there an account serial number?
     if query_string.get("sn"):
         account_serial_number = query_string["sn"][0]

